lib/protobuf/field/varint_field.rb

```diff
@@ -32,6 +42,11 @@ def self.encode(value)
         (bytes << value).pack('C*')
       end
 
+      # Load the cache of VarInts on load of file
+      (0..CACHE_LIMIT).to_a.each do |cached_value|
```

Comment:
maybe a little late, but ranges are iterable without needing the `.to_a`, the `.to_a` ends up creating a medium-sized array that's not really used